azure-functions-sdk/src/commands/new/blob.rs

```diff
@@ -42,7 +48,10 @@ impl<'a> Blob<'a> {
 impl<'a> From<&'a ArgMatches<'a>> for Blob<'a> {
     fn from(args: &'a ArgMatches<'a>) -> Self {
         Blob {
-            name: args.value_of("name").unwrap(),
+            name: args.value_of("positional-name").unwrap_or_else(|| {
+                args.value_of("name")
+                    .unwrap_or("Default fallback - never reached")
```

Comment:
Can this be an `expect` instead?

Reply:
If so, this comment applies to each command.

Reply:
Hi @ajnirp.  Sorry for the delay in following up with you.  I'd like to get this merged soon.  Did you see this review question, namely changing `unwrap_or` to `expect`?  Thanks!

Reply:
Done!

Reply:
The CI checks for `cargo fmt -- --check` and `cargo clippy --release -- --Dwarnings` are failing, but those same commands are working (i.e. `$?` is 0) for me locally. Any idea what I might be missing?

Reply:
The clippy error is from generated code that we can ignore for now.

The rustfmt one is odd.  The CI uses the latest stable, so if you're also using the latest stable I can't explain the difference.

Let me look into this tonight if I can; I won't block merging your PR for a format / linter error, though.

Reply:
Sounds good, thanks!